libevmasm/SourceLocation.h

```diff
@@ -79,20 +83,24 @@ inline std::ostream& operator<<(std::ostream& _out, SourceLocation const& _locat
 bool SourceLocation::operator<(SourceLocation const& _other) const
 {
 	if (!sourceName || !_other.sourceName)
+	{
+		if (!sourceName && !_other.sourceName)
+			return std::make_tuple(start, end) < std::make_tuple(_other.start, _other.end);
```

Comment:
Can this and the next line be shortened to
`return std::make_tuple(!!sourceName, start, end) < std::make_tuple(!!_other.sourceName, _other.start, _other.end);`?